packages/opentelemetry-api/src/trace/span_context.ts

```diff
@@ -49,7 +49,7 @@ export interface SpanContext {
    *
    * SAMPLED = 0x1 and NONE = 0x0;
    */
-  traceFlags: TraceFlags;
+  traceFlags?: TraceFlags;
```

Comment:
I'm sorry if I missed it in an earlier review conversation, but what is the reason for this API change? Traceflags is not an optional part of the span context and I would not expect to have to change that just to propagate a couple additional data fields from b3.

Reply:
Cool. I can change this back, but the reason I did this is that in b3, if there's no sampled value passed in the header it means to defer the decision, but what we were doing was just setting it to 0, which I understand as not sampling.

Reply:
I don't think we should change our API for that reason. If you need to implement some other sampling behavior, then the correct way would be to implement a b3 sampler which handles this delegated functionality.

Reply:
ohh, okay. I think it's best to do that in a separate issue because the scope of this issue was to just get parentSpanId and debug propagated through. I'll change of the other stuff back. Thanks.